hw_05/Program.cs

```diff
@@ -0,0 +1,147 @@
+﻿using System;
+
+namespace hw_05
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Task 1:");
+            Task1();
+            Console.WriteLine();
+
+            Console.WriteLine("Task 2:");
+            Task2();
+            Console.WriteLine();
+
+            Console.WriteLine("Task 3:");
+            Task3();
+            Console.WriteLine();
+
+            Console.Read();
+        }
+
+        public static void Task1()
+        {
+            int arrSize = 5;
+
+            int[] randomArr = FillArrayRandom(new int[arrSize], 0, arrSize);
+
+            Console.WriteLine($"Enter {arrSize} integer to fill an array");
+            int[] userArr = FillArrayByUser(new int[arrSize], 0, arrSize);
+
+            int[] resultArr = new int[arrSize];
+            for (int i = 0; i < arrSize; i++)
+            {
+                resultArr[i] = userArr[i] + randomArr[i];
+            }
+
```

Comment:
I would also make a separate method to fill the result array out like you have done for both cases above.

Reply:
done